Package.swift

```diff
@@ -1362,7 +1362,7 @@ func googleAppMeasurementDependency() -> Package.Dependency {
     return .package(url: appMeasurementURL, branch: "main")
   }
 
-  return .package(url: appMeasurementURL, exact: "10.23.0")
```

Comment:
Please revert this. Now that 10.23.0 is published, the workaround is not necessary.